staging/src/k8s.io/apiserver/pkg/cel/mutation/common/val.go

```diff
@@ -104,16 +108,33 @@ func (v *ObjectVal) IsZeroValue() bool {
 // convertField converts a referred ref.Val to its expected type.
 // For objects, the expected type is map[string]any
```

Comment:
nit: the godoc mentions objects being map[string]any but there is no specific case for them, im not sure if that is implicitly somewhere but I just expected to see an explicit case/comment for that case

Reply:
Thank you for the reviews. Objects are covered by the default `v.Value()` case. The implementation is above this function but the diff collapsed it.